custom_background/models/report.py

```diff
@@ -239,111 +216,36 @@ def get_bg_per_lang(self):
         custom_background = custom_bg_lang[:1].background_pdf
         return custom_background
 
+    def _build_wkhtmltopdf_args(
+        self, paperformat_id, landscape, specific_paperformat_args=None, set_viewport_size=False
+    ):
+        command_args = super()._build_wkhtmltopdf_args(
+            paperformat_id, landscape,
+            specific_paperformat_args=specific_paperformat_args,
+            set_viewport_size=set_viewport_size,
+        )
+        command_args.extend(['--print-media-type'])
```

Comment:
I am not sure why it will be needed here.
If it should be generic, it should land in Odoo core.
@wtaferner

Reply:
@bizzappdev 
I agree, this should land in Odoo core, I will give it a shot, but as the scss is modifying the screen layout too, it is the quick and easy way to not f*** up the Core layout by making body background transparent.